libclc/amdgcn-amdhsa/libspirv/atomic/atomic_sub.cl

```diff
@@ -18,7 +18,7 @@
           enum MemorySemanticsMask semantics, TYPE val) {                                                                     \
```

Comment:
The changes to this file should be exported to a separate PR, they also deserve an explanation for the sign flipping.

Reply:
Will do